render/pod.go

```diff
@@ -57,6 +57,34 @@ var PodServiceRenderer = ConditionalRenderer(renderKubernetesTopologies,
 	)),
 )
 
+// DeploymentRenderer is a Renderer which produces a renderable kubernetes deployments
+// graph by merging the pods graph and the deployments topology.
+var DeploymentRenderer = ApplyDecorators(
+	FilterEmpty(report.Pod,
+		MakeReduce(
+			MakeMap(
+				MapPod2Deployment,
+				PodRenderer,
+			),
+			SelectDeployment,
+		),
+	),
```

Comment:
The selectors on a deployment match either. Technically a deployment manages a replicaset which manages the pods, but in practice deployments and replica sets are somewhat interchangeable (with the difference that they tell you not to use replica sets). However, showing the full hierarchy may be more clear.